Adyen.Test/HeaderRequestTest.cs

```diff
@@ -30,90 +33,80 @@ namespace Adyen.Test
     [TestClass]
     public class HeaderRequestTest
     {
-
-        private readonly HttpUrlConnectionClient _httpUrlConnectionClient;
+        
         private readonly string _endpoint = "https://endpoint:8080/";
-
-        public HeaderRequestTest()
-        {
-            _httpUrlConnectionClient = new HttpUrlConnectionClient();
-        }
+        
         [TestMethod]
         public void BasicAuthenticationHeadersTest()
         {
-            var httpWebRequest = _httpUrlConnectionClient.GetHttpWebRequest(_endpoint, MockPaymentData.CreateConfingMock(), false);
-
-            Assert.IsNotNull(httpWebRequest.UserAgent);
```

Comment:
Doing a quick before/after check: I see 2 (potentially 3) mismatches in left-out checks. Just checking if it was done intentionally.

- [x] Assert.IsNotNull(httpWebRequest.UserAgent);

- [x] Assert.AreEqual(httpWebRequest.Address.ToString(), _endpoint);
 Assert.AreEqual<string>(httpRequestMessage.RequestUri?.ToString(), _endpoint); // I assume this is the same :)

- [x] Assert.AreEqual(httpWebRequest.Headers["Accept-Charset"], "UTF-8");

- [x] Assert.AreEqual(httpWebRequest.Headers["Cache-Control"], "no-cache");

- [ ] Assert.AreEqual(httpWebRequest.ContentType, "application/json");
Missing (?)

- [x] Assert.IsNotNull(httpWebRequest.Headers["Authorization"]);

- [ ] Assert.IsTrue(httpWebRequest.UseDefaultCredentials);
Missing

- [x] Assert.AreEqual(httpWebRequest.Headers["Cache-Control"], "no-cache");

- [ ] Assert.IsNull(httpWebRequest.Headers["x-api-key"]);
Missing (?)

Reply:
Fixed it